inc/namespace.php

```diff
@@ -196,3 +198,77 @@ function enqueue_scripts() {
 	// Print queued scripts.
 	print_head_scripts();
 }
+
+/**
+ * Schedule index maintenance daily.
+ */
+if( ! wp_next_scheduled( __NAMESPACE__ . '\\index_maintenance' ) ) {
+	wp_schedule_event( time(), 'daily',  __NAMESPACE__ . '\\index_maintenance' );
+}
+
+
+/**
+ * Delete old Analytics indexes in Elasticsearch.
+ */
+function delete_old_indexes() : ?array {
+	// Index age in days.
```

Comment:
For filters we can document them like we would a function, so a docblock with params and an explanation. This will help us in future as we can generate documentation from these:
```suggestion
	/**
	 * Filter the maximum index age in days.
	 *
	 * @param int $days The maximum number of days to keep indexes for, defaults to 7.
	 */
```